lib/rules/define-macros-order.js

```diff
@@ -11,13 +11,13 @@ const MACROS_PROPS = 'defineProps'
 const MACROS_OPTIONS = 'defineOptions'
 const MACROS_SLOTS = 'defineSlots'
 const MACROS_MODEL = 'defineModel'
-const ORDER_SCHEMA = [
+const ORDER_SCHEMA = new Set([
```

Comment:
Maybe rename to `KNOWN_MACROS` or similar? It's not used in the schema anymore.

Also, please add `defineExpose` (via a new constant `MACROS_EXPOSE`) to the set.